Workiva/json_schema
# Make it very easy to add custom json schema tests.

## Ultimate problem:
There is a hoop to jump through for creating json-schema spec tests.

## How it was fixed:
Update the generator to look at an additional directory when generating the test file.

## Testing suggestions:
1. Create a new test that should fail.
2. Run `make gen-fixtures`
3. Run `make test` and see some fresh failing tests.
4. Fix the test and run `make gen-fixtures` again.
5. Run the tests and see them all pass.


## Potential areas of regression:



---

> __FYA:__ @michaelcarter-wf
